### ISSUE TITLE
Environment configuration

### ISSUE THREAD

**Issue by chaowentao** [L1aoXingyu/fcn.pytorch#1]
Could you please provide detailed environment configuration description and completed training process description (including data sets download, etc.)? Thanks a lot~

**Comment by L1aoXingyu** [comment]
I am sorry that I forgot about the dataset parts. I will add this as soon as possible. Btw, for environment configuration, you only need to install the package I list in requirements.

**Comment by chaowentao** [comment]
Thanks  a lot. I am your students from cs231n camp. Thanks again.

**Comment by L1aoXingyu** [comment]
I have added the dataset configuration. Hope this can help.

**Comment by chaowentao** [comment]
I reconfigured the environment again, the result is still the original problem. I guess it may be a problem with the windows system, then I will try it under the Ubuntu system. Thank  you.
![_20181207143454](https://user-images.githubusercontent.com/37093663/49631515-48ffdc00-fa2d-11e8-93c2-cefb09773295.png)

**Comment by L1aoXingyu** [comment]
you can try to set num_workers=0 in `DataLoader`

**Comment by L1aoXingyu** [comment]
Can you run this training script? If you have any other problems, you can tell me in this issue and in case others meet the same problems, too.

**Comment by Merofine** [comment]
Thanks.i meet some problem.and it work now.Thanks.

**Comment by Merofine** [comment]
but it is too slow.
![捕获](https://user-images.githubusercontent.com/33657760/54487069-171fd080-48cc-11e9-8c2b-36b76db50f81.PNG)
 @L1aoXingyu

**Comment by L1aoXingyu** [comment]
It is very slow because the batch size is 1, you can set the batch size = 32, then do some preprocessing, such as center crop etc, to make the input image size same.

**Comment by Merofine** [comment]
Yes, I changed it to 32, but it still reported a mistake. It is estimated that there is no pretreatment. Actually, I'm particularly interested in your project, but I'm just a whitewash. Could you please give me more detailed steps? 
![捕获](https://user-images.githubusercontent.com/33657760/54502432-80075700-4965-11e9-9d64-cde9264243bf.PNG)

**Comment by KinsoZHENG** [comment]
> Yes, I changed it to 32, but it still reported a mistake. It is estimated that there is no pretreatment. Actually, I'm particularly interested in your project, but I'm just a whitewash. Could you please give me more detailed steps?
> ![捕获](https://user-images.githubusercontent.com/33657760/54502432-80075700-4965-11e9-9d64-cde9264243bf.PNG)
Hi, I've meet the same problem. Have you fixed it? 
Thanks a lot.

**Comment by KinsoZHENG** [comment]
> > Yes, I changed it to 32, but it still reported a mistake. It is estimated that there is no pretreatment. Actually, I'm particularly interested in your project, but I'm just a whitewash. Could you please give me more detailed steps?
> > ![捕获](https://user-images.githubusercontent.com/33657760/54502432-80075700-4965-11e9-9d64-cde9264243bf.PNG)
> > Hi, I've meet the same problem. Have you fixed it?
> > Thanks a lot.

I have tried to fix the batch size problem. After changing the batch size in config/defaults.py, 
please add         
       img = img.resize((x, x)) # the size what you wanna
after https://github.com/L1aoXingyu/fcn.pytorch/blob/7f592407f41325375baff5b3514567fb4c5f9a62/data/datasets/voc.py#L34
        label = label.resize((x, x)) # the same size of before
after
https://github.com/L1aoXingyu/fcn.pytorch/blob/7f592407f41325375baff5b3514567fb4c5f9a62/data/datasets/voc.py#L36
It can make the program work.
